Update monitoring-archive-activity-log.md

vnatv opened the pull request:
The max number of days, which possible to enter is 365, when using Azure portal option.

azurecla commented:
Hi __@bernitorres__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (betorres). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

bernitorres commented:
Sorry, I made a mistake. I will open this PR against private

Commit:
Update monitoring-archive-activity-log.md

The max number of days, which possible to enter is 365, when using Azure portal option.
